docs/content/concepts/automation/schedules/troubleshooting.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: "Troubleshooting schedules | Dagster Docs"
+---
+
+# Troubleshooting schedules
+
+Run into issues with a schedule? Use this guide to diagnose and resolve the problem.
+
+---
+
+## Step 1: Verify the schedule is included in the Definitions object
+
+First, verify that the schedule has been included in a <PyObject object="Definitions" /> object. This ensures that the schedule is detectable and loadable by Dagster tools like the Dagster UI and CLI:
+
+```python
+defs = Definitions(
+   assets=[asset_1, asset_2],
+   jobs=[job_1],
+   schedules=[all_assets_job_schedule],
+)
+```
+
+Refer to the [Code locations documentation](/concepts/code-locations) for more info.
+
+---
+
+## Step 2: Verify that the schedule has been started
+
+1. In the Dagster UI, click **Overview > Schedules tab**.
```

Comment:
For me, in Purina, this was under "Automation -> Schedules"
<img width="797" alt="image" src="https://github.com/dagster-io/dagster/assets/5807118/9af94f4f-739e-450f-a6d3-47b6f61c256e">

This is with Experimental Navigation turned on:
<img width="573" alt="image" src="https://github.com/dagster-io/dagster/assets/5807118/2169f525-0573-4e1b-893e-12c547cf5e98">

Reply:
Right - I'm thinking not to use the experimental nav as it may change. Once it's final I'll go back and update things